praw/exceptions.py

```diff
@@ -81,6 +142,33 @@ def parse_exception_list(
             )
             for exception in exceptions
         ]
+        for exception in baselist:
+            exception._called_from_reddit_api_exception = True
+        return baselist
+
+    @classmethod
+    def parse_exception_list(
+        cls, exceptions: List[Union[RedditErrorItem, List[str]]]
+    ) -> List[RedditErrorItem]:
+        """Covert an exception list into a :class:`.RedditErrorItem` list.
+
+        .. warning:: This method is deprecated. Use the private method of the
```

Comment:
I don't think we should recommend using the private method. We could instead say something like this method is deprecated and will be removed without replacement in PRAW 8.0.